components/announcement_bar/__snapshots__/text_dismissable_bar.test.jsx.snap

```diff
@@ -1,119 +1,11 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`components/TextDismissableBar should match snapshot 1`] = `
-<AnnouncementBar
-  color=""
-  extraProp="test"
-  handleClose={[Function]}
-  message={
-    <Connect(Markdown)
-      message="sample text"
-      options={
-        Object {
-          "mentionHighlight": false,
-          "singleline": true,
-        }
-      }
-    />
-  }
-  onDismissal={[MockFunction]}
-  showCloseButton={true}
-  textColor=""
-  type="critical"
-/>
-`;
+exports[`components/TextDismissableBar should match snapshot 1`] = `""`;
```

Comment:
It looks like these snapshots were broken. Do the tests for this need to be updated or is there something wrong here? [This line](https://github.com/mattermost/mattermost-webapp/pull/5291/files#diff-5d31096ca67f4f64a81f8a0587adf9c7R54) looks like it might be causing this.

Reply:
Oops